index.html

```diff
@@ -0,0 +1,263 @@
+      </div>
+      <div class="items1">
+        <h2><a href="#">Books</a></h2>
+        <p><a href="#">Physical & digital books</a></p>
+      </div>
+      <div class="items1"><i class="fa fa-search"></i> Topics</div>
+    </nav>
+
+    <div class="articles flex lower-bar">
+      <p class="latest-articles">Don’t Miss These Latest Articles</p>
+      <div class="items3">
+        <h1>Art Direction For The Web Using CSS Shapes</h1>
+      </div>
+      <div class="items2">
+        <h1>Privacy UX: Better Cookie Consent Experiences</h1>
+      </div>
+      <div class="items3">
+        <h1>Understanding Subresource Integrity</h1>
+      </div>
+      <div class="items2">
+        <h1>Digging Into The Display Property: The Two Values Of Display</h1>
+      </div>
+    </div>
+    <div class="membership top-bar items4"><h1>Membership</h1></div>
+
+    <div class="latest-posts flex lower-bar">
+      <div class="items3">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In faucibus
+          faucibus
+        </p>
+      </div>
+      <div class="items2">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In faucibus
+          faucibus eget molestie elit nibh ac odio.
+        </p>
+      </div>
+      <div class="items1">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In faucibus
+          faucibus eget molestie elit nibh ac odio.
+        </p>
+      </div>
+      <div class="items1">
+        <h1>Advert space</h1>
+      </div>
+
+      <div class="large-boxes items4"><h1>Ad to SignUp</h1></div>
+      <div class="items3">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum lacus at tempus. Vivamus dapibus consectetur
+          tincidunt. Donec quis posuere velit, vel congue elit. In fus velit
+          ultricies neque, eget molestie elit nibh ac odio.
+        </p>
+      </div>
+      <div class="items4">
+        <h2>Lorem Ipsum</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+          condimentum fermentum t. Pellentesque eleifend sed magna eget
+          sollicitudin. Aenean pellentesque, augue non dictum placerat, tellus
+          velit ultricies neque, eget molestie elit nibh ac odio.
+        </p>
+      </div>
+      <div class="items3">
```

Comment:
You are using `<div>` tags for these articles, but there are better options to address these circumstances. (For example `<article>` . Read more about [HTML semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) elements and implement it in your project.